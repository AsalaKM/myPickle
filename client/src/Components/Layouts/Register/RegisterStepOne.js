import React from "react"

import { Intro, Button } from "./Register.style"

export default class RegisterIntro extends React.Component {
  render() {
    console.log(this.props)
    return (
      <Intro>
        <h2 className="tc mp-primary-color">Wellness Areas</h2>
        <p>
          So we can customize your profile, please select the area(s) of wellness where you provide
          support.
        </p>
        <div className="flex items-center justify-between w-100">
          <Button onClick={this.props.handlePrevious}>Back</Button>
          <Button onClick={this.props.handleNext}>Next</Button>
        </div>
      </Intro>
    )
  }
}
