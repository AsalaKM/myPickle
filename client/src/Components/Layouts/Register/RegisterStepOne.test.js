import React from "react"
import renderer from "react-test-renderer"
import { StaticRouter } from "react-router-dom"

import RegisterStepOne from "./RegisterStepOne"

test("Register matches snapshot", () => {
  const context = {}
  const component = renderer.create(
    <StaticRouter context={context}>
      <RegisterStepOne />
    </StaticRouter>
  )
  expect(component).toMatchSnapshot()
})
