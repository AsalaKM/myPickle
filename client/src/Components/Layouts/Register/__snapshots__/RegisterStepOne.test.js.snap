// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register matches snapshot 1`] = `
Array [
  <div
    className="sc-bdVaJa flex flex-column items-center ph3 NjJhJ"
  >
    <h2
      className="tc mp-primary-color"
    >
      Wellness Areas
    </h2>
    <p>
      So we can customize your profile, please select the area(s) of wellness where you provide support.
    </p>
  </div>,
  <div
    className="sc-EHOje bCBdnp"
  >
    <div
      className="sc-htpNat mp-b--primary-color klKWzH"
    />
    <div
      className="sc-htpNat sc-bxivhb mp-b--primary-color dzjdHN"
    />
    <div
      className="sc-htpNat sc-ifAKCX mp-b--primary-color jvwKmB"
    />
  </div>,
  <div
    className="flex items-center justify-between w-100"
  >
    <button
      className="sc-bwzfXH mp-bg-white mp-b--primary-color kPAatI"
    >
      Back
    </button>
    <button
      className="sc-bwzfXH mp-bg-white mp-b--primary-color kPAatI"
    >
      Next
    </button>
  </div>,
]
`;
